sdk/python/kfp/dsl/_container_op.py

```diff
@@ -1017,6 +1018,7 @@ def add_pod_label(self, name: str, value: str):
     def set_retry(self,
                   num_retries: int,
                   policy: Optional[str] = None,
+                  expression: Optional[str] = None,
```

Comment:
We need to add this to the end, so it won't break users who call this method using sequential arguments.
Otherwise lgtm.